### PR TITLE
[ngtcp2] Update to 1.5.0

### PR DESCRIPTION
**Pull request by alagoutte**
Disable Testing (via munit)
Remove build cunit parameter
Update no export patch

### CODE REVIEW

**Comment by alagoutte** [comment]
Not yet ready for merge (no yet 1.5.0 release yet)

**Comment by talregev** [comment]
@alagoutte Thank you for your update!
Why don't you export ngtcp2 in cmake official? instead to change the patch here?
Also, any plan to add openssl support, now it release on 3.3.0, and if not, what is missing?

**Comment by alagoutte** [comment]
> @alagoutte Thank you for your update! Why don't you export ngtcp2 in cmake official? instead to change the patch here? Also, any plan to add openssl support, now it release on 3.3.0, and if not, what is missing?

I'm not author of ngtcp2 ;-)
I'm only update the package...

I can look for see to include export on ngtcp2 

and for openssl, it is not planned -> https://github.com/ngtcp2/ngtcp2/issues/990

**Comment by alagoutte** [comment]
Updated with new release

**Comment by dg0yt** [comment]
> and for openssl, it is not planned -> [ngtcp2/ngtcp2#990](https://github.com/ngtcp2/ngtcp2/issues/990)

I wonder if that answer also covers OpenSSL 3.3  and later version.

File: ports/ngtcp2/portfile.cmake [hunk 1]
```diff
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ngtcp2/ngtcp2
     REF "v${VERSION}"
-    SHA512 3d266b80fbc44bf0711f4791db4787d80c989beb84ff4fe33e0f8fd25b8bb6d6d768bcbf990ca4bde0bcbaad5c7566bfb6a431b5bbec844247d8a4b417d451d9
+    SHA512 8be12732e27eaabfbedae83ceee66ae0a269edebc614a5dd7645b8019d1f394be91391943baade93bad71e6100e22e46240b2b01c97ba3dedd7ef343113351e1
     HEAD_REF master
```
Comment (Cheney-W) [inline ports/ngtcp2/portfile.cmake:6]:
```suggestion
    HEAD_REF main
```
The default branch of `ngtcp2` is main.

**Comment by talregev** [comment]
@alagoutte 

```
1.5.0 is declared with c529fdf592d945696a24e6cb0f7f169aea71e0c2, but the local port has a different SHA e84533ed07e21e33b38553c3b75abd9a65536e76.
Please update the port's version fields and then run:
vcpkg x-add-version ngtcp2 --overwrite-version
git add versions
git commit -m "Update version database"
to add the new version.
To attempt to resolve all errors at once, run:
vcpkg x-add-version --all

```

**Review by Cheney-W — APPROVED** [review]

**Review by data-queue — APPROVED** [review]

**Pull request merged**